### PR TITLE
add my folder and readme

### PR DESCRIPTION
**Pull request by DVDripXvid**
i want merge to your branch

### CODE REVIEW

File: oliver.rottenhoffer/readme.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+az �let sz�p
+j�v�h�ten h�sv�t van.
+kir�ly
```
Comment (marcelldeak) [inline oliver.rottenhoffer/readme.txt]: jó lesz

Comment (DVDripXvid) [inline oliver.rottenhoffer/readme.txt]: köszi

**Comment by marcelldeak** [comment]
ez mi?

**Comment by DVDripXvid** [comment]
nem tom

**Comment by marcelldeak** [comment]
had menjen :D

**Comment by DVDripXvid** [comment]
had szóljon

**Pull request merged**